src/Gateway.tsx

```
@@ -2,21 +2,33 @@ import { doc, getDoc } from "firebase/firestore";
+/**
+ * Loader function for the Gateway component.
+ * @param {Object} context - The context object.
+ * @param {Object} context.params - The parameters object.
+ * @param {string} context.params.id - The ID of the vote.
+ * @returns {Promise<Response | void>} - A promise that resolves to a Response object or void.
+ */
+Gateway.loader = async function loader({
+  params,
+}: {
+  params: { id: string };
+}): Promise<Response | void> {
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)